modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -9,11 +9,20 @@
 
 {product-title} includes link:https://kubernetes.io/docs/concepts/security/pod-security-admission[Kubernetes pod security admission]. Globally, the `privileged` profile is enforced, and the `restricted` profile is used for warnings and audits.
 
+[NOTE]
+====
+If you want to preview _enforcing_ the `restricted` profile on your non-production cluster as a Technology Preview, you can enable the `TechPreviewNoUpgrade` feature set. If you enable the `TechPreviewNoUpgrade` feature set, pods are rejected if they violate pod security standards, instead of only logging a warning.
+
+Pod security admission restricted enforcement is only activated if you enable the `TechPreviewNoUpgrade` feature set after your {product-title} cluster is installed. It is not activated if you enable the `TechPreviewNoUpgrade` feature set during cluster installation.
```

Comment:
Is this true for 4.13 also? I imagine not; but wanted to ask.

Reply:
Nope, only for 4.12. Thanks for checking though!